lib/react.dart

```diff
@@ -714,6 +714,36 @@ abstract class Component2 implements Component {
   /// See: <https://facebook.github.io/react/docs/react-component.html#unmounting-componentwillunmount>
   void componentWillUnmount() {}
 
+  /// ReactJS lifecycle method that is invoked after an error is thrown by a
+  /// descendant.
+  ///
+  /// Use this method primarily for logging errors, but because it takes
+  /// place after the commit phase side-effects are permitted.
+  ///
+  /// __Note__: This method, along with [getDerivedStateFromError] will only
+  /// be called if `skipMethods` in [_registerComponent] is overridden with
```

Comment:
Yep, that's what it should have been. I was meaning to reference `_registerComponent` in `react_client.dart` because the private methods in `react_client` are ultimately what get used, but it should definitely be `registerComponent2`.